src/Scanner.debug.js

```diff
@@ -34,12 +38,81 @@ const mockInterface = {
     },
     attachTooltip (...args) {
         console.log('device.attachTooltip', args)
+    },
+    isTooltipActive: () => {
+        return false
+    },
+    get scanner () {
+        return state.scanner
     }
 }
 
-// @ts-ignore
-const scanner = createScanner(mockInterface, {
-    initialDelay: 1 // allow debugging directly on macOS - if this was 0 then it would try to use requestIdleCallback, which is absent in WebKit
-})
+let state = {
+    /** @type {import('./Scanner.js').Scanner | undefined} */
+    scanner: undefined,
+    /** @type {HTMLSelectElement|null} */
+    list: document.querySelector('select[name="html-list"]')
+}
+
+const url = new URL(window.location.href)
+const initial = url.searchParams.get('form')
+const log = url.searchParams.has('log')
 
-scanner.init()
+if (log) {
+    sessionStorage.setItem('ddg-autofill-debug', 'true')
+    sessionStorage.setItem('ddg-autofill-perf', 'true')
+} else {
+    sessionStorage.setItem('ddg-autofill-debug', 'false')
+    sessionStorage.setItem('ddg-autofill-perf', 'false')
+}
+
+loadList()
+    .then(() => {
+        if (initial) {
+            loadNewForm(initial).catch(console.error)
+        }
+    })
+
+async function loadList () {
+    const url = new URL(`/test-forms/index.json`, window.location.href)
+    fetch(url)
+        .then(response => response.json())
+        .then(x => {
+            x.forEach(item => {
```

Comment:
Just a nicer name 🙂.

```suggestion
        .then(testForms => {
            testForms.forEach(item => {
```